service/src/main/resources/templates/ehr_encounter_to_ehr_composition_template.mustache

```diff
@@ -1,19 +1,31 @@
 <component typeCode="COMP">
     <ehrComposition classCode="COMPOSITION" moodCode="EVN">
         <id root="{{encounterStatementId}}" />
```

Comment:
Can you make sure the elements you are adding are in the correct order according to the MiM? The XSDs for all of the HL7 are strictly ordered.

Reply:
Seems okay according to https://data.developer.nhs.uk/dms/mim/4.2.00/Domains/GP2GP/Tabular%20View/RCMR_HD030100UK04-NoEdit.htm#EhrComposition